Automattic/simplenote-ios
# Supporting accent characters in Search

### Fix
In this PR we're addressing an (ancient) issue: we're turning the Search Predicate into Diacritic Insensitive.

This means that: from now on, Search will be "accent insensitive". For instance, if a note contains the word `esdrújula`, and the keyword is `esdrujula` (that is: lacks the accent), it's expected to show up in the search results.

Closes #9

Thank you @bummytime !!

### Test
1. Launch the app
2. Add a new note with the word `esdrújula` and the tag `jamás`
3. Press back

- [x] Verify that the word `esdrujula` (without accent) yields the note you've just added
- [x] Verify that the word `jamas` yields the tag `jamás` (with accent)

### Release
`RELEASE-NOTES.txt` was updated in f377d28 with:

> Search is now diacritic insensitive: type search terms with or without accents to get matches!
